src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDownForced.groovy

```diff
@@ -9,15 +9,28 @@ import org.gradle.util.VersionNumber
 class ComposeDownForced extends DefaultTask {
     ComposeSettings settings
 
+    List<String> dependentServices = [] // for testing
```

Comment:
Please remove this field - it's IMHO bad to have _testing stuff_ in production code.

In the test, you could e.g. execute `docker-compose ps` to list running containers. Then you can check that dependent container are not present there.

Reply:
That seems reasonable. I saw another field that had this so I figured that was the only way. I’ll make these changes.